AirCasting/Utils/NetworkChecker.swift

```diff
@@ -0,0 +1,22 @@
+// Created by Lunar on 08/07/2021.
+//
+
+import Foundation
+import Network
+
+class NetworkChecker {
```

Comment:
would be nice to have it protocolized so we can change implementation to a mock one while unit testing stuff